### ISSUE TITLE
Worst case? No Access to e2e encrypted Passwords after last update of NC and the daily Build of Passwords

### ISSUE THREAD

**Issue by huddx01** [marius-wieschollek/passwords#342]
### System Information
**Server:**
```json
 Run "php ./occ passwords:system:report debug" in your Nextcloud install directory and put the output here
 or fill out the fields below
    "version": {
        "server": "20.0.7.1",
        "app": "2021.2.0-build3937",
        "php": "7.4.14"
    },
    "environment": {
        "os": "Linux",
        "architecture": "x86_64",
        "bits": 64,
        "database": "mysql",
        "cron": "cron",
        "proxy": false,
        "sslProxy": false,
        "subdirectory": false
    },
    "legacyApi": {
        "enabled": 2,
        "used": false
    },
    "services": {
        "images": "imagick",
        "favicons": "local",
        "previews": "wkhtml",
        "security": "bigdb",
        "words": "local",
        "previewApi": false,
        "faviconApi": false
    },
    "settings": {
        "channel": "stable",
        "nightlies": false,
        "handbook": false,
        "performance": 5
    },
    "encryption": {
        "sse": {
            "SSEv1r1": false,
            "SSEv1r2": true,
            "SSEv2r1": true,
            "none": true,
            "default": "SSEv2r1"
        },
        "cse": {
            "CSEv1r1": true,
            "none": true,
            "default": "CSEv1r1"
        }
    }
}

**Client:**
```
 Browser and Version: current Safari on OSX and iOS
 Client OS and Version: OSX, iOS
```

### Steps to reproduce
1. <!-- Describe PRECISELY and DETAILED the necessary steps to reproduce the bug -->
2. <!-- Provide sample data if needed -->
3. <!-- Include relevant user settings and app settings if not standard -->

### Expected result
What should happen?
Passwords should appear after Login with my Account, then klick on Passwords Tab, then logged on with e2e Password and TOTP Token
### Actual result
Only shared (not encrypted Accounts/Passwords) are shown.
My encrypted Folders and the Accounts/Passwords are not not shown. A message: "Unable to complete request" is shown on the top-right site in NC-Passwords App in Browser.

### Nextcloud log
<details>
<summary>Nextcloud log</summary>

{"reqId":"xxx","level":4,"time":"2021-02-08T21:13:32+01:00","remoteAddr":"xxx","user":"xxx","app":"passwords","method":"POST","url":"/index.php/apps/passwords/api/1.0/folder/show","message":{"Exception":"Exception","Message":"Typed property OCA\\Passwords\\Encryption\\Object\\SseV2Encryption::$isAvailable must not be accessed before initialization","Code":0,"Trace":[{"file":"xxx/nextcloud/lib/private/AppFramework/App.php","line":152,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"xxx/nextcloud/lib/private/Route/Router.php","line":309,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"xxx/nextcloud/lib/base.php","line":1008,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"xxx/nextcloud/index.php","line":37,"function":"handleRequest","class":"OC","type":"::"}],"File":"xxx/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","Line":110,"Previous":{"Exception":"Error","Message":"Typed property OCA\\Passwords\\Encryption\\Object\\SseV2Encryption::$isAvailable must not be accessed before initialization","Code":0,"Trace":[{"file":"xxx/nextcloud/apps/passwords/lib/Services/EncryptionService.php","line":203,"function":"isAvailable","class":"OCA\\Passwords\\Encryption\\Object\\SseV2Encryption","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Services/Object/AbstractRevisionService.php","line":180,"function":"getDefaultEncryption","class":"OCA\\Passwords\\Services\\EncryptionService","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Services/Object/FolderRevisionService.php","line":226,"function":"getSseEncryption","class":"OCA\\Passwords\\Services\\Object\\AbstractRevisionService","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Services/Object/FolderRevisionService.php","line":103,"function":"createModel","class":"OCA\\Passwords\\Services\\Object\\FolderRevisionService","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Services/Object/FolderRevisionService.php","line":71,"function":"getBaseRevision","class":"OCA\\Passwords\\Services\\Object\\FolderRevisionService","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Helper/ApiObjects/AbstractObjectHelper.php","line":137,"function":"findByUuid","class":"OCA\\Passwords\\Services\\Object\\FolderRevisionService","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Helper/ApiObjects/FolderObjectHelper.php","line":93,"function":"getRevision","class":"OCA\\Passwords\\Helper\\ApiObjects\\AbstractObjectHelper","type":"->"},{"file":"xxx/nextcloud/apps/passwords/lib/Controller/Api/AbstractObjectApiController.php","line":145,"function":"getApiObject","class":"OCA\\Passwords\\Helper\\ApiObjects\\FolderObjectHelper","type":"->"},{"file":"xxx/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":169,"function":"show","class":"OCA\\Passwords\\Controller\\Api\\AbstractObjectApiController","type":"->"},{"file":"xxx/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":100,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"xxx/nextcloud/lib/private/AppFramework/App.php","line":152,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"xxx/nextcloud/lib/private/Route/Router.php","line":309,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"xxx/nextcloud/lib/base.php","line":1008,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"xxx/nextcloud/index.php","line":37,"function":"handleRequest","class":"OC","type":"::"}],"File":"xxx/nextcloud/apps/passwords/lib/Encryption/Object/SseV2Encryption.php","Line":102},"CustomMessage":"Typed property OCA\\Passwords\\Encryption\\Object\\SseV2Encryption::$isAvailable must not be accessed before initialization"},"userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.3 Safari/605.1.15","version":"20.0.7.1","id":"xxx"}
</details>

### Browser log
<details>
<summary>Browser log</summary>

```
not really necessary, tried in Safari and Firefox...
```
</details>

**Comment by marius-wieschollek** [comment]
Can you check if this is fixed with [build 3939](https://apps.nextcloud.com/apps/passwords/releases)?

**Comment by huddx01** [comment]
> Can you check if this is fixed with [build 3939](https://apps.nextcloud.com/apps/passwords/releases)?

Perfectly! It works again now..

Thank you.

**Comment by marius-wieschollek** [comment]
i published the fix with 2021.1.3
